api/src/main/java/io/opencensus/tags/propagation/TagContextTextSerializer.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.tags.propagation;
+
+import io.opencensus.tags.TagContext;
+
+/**
+ * Object for serializing and deserializing {@link TagContext}s with HTTP text format.
+ *
+ * <p>OpenCensus uses W3C Correlation Context as the HTTP text format. For more details, see <a
+ * href="https://github.com/w3c/correlation-context">correlation-context</a>.
+ *
+ * @since 0.21
+ */
+public abstract class TagContextTextSerializer {
```

Comment:
In this approach you force tags to be propagated in one string. What if we want to propagate them in multiple strings? Take a look at the trace part:
https://github.com/census-instrumentation/opencensus-java/blob/master/api/src/main/java/io/opencensus/trace/propagation/TextFormat.java

Reply:
OK make sense to me, added support for encoding in multiple strings in https://github.com/census-instrumentation/opencensus-java/pull/1800/commits/56573558e5c861280d6e984d412f8d6c80b97933.